packages/types/package.json

```diff
@@ -15,11 +15,10 @@
     "src/global.d.ts",
     "/dist"
   ],
-  "peerDependencies": {
-    "@polkadot/api": "^9"
+  "dependencies": {
+    "bn.js": "5.2.1"
```

Comment:
I think we need `near-api-js` as  a dependency for JsonRPCProvider.

Is bn.js a dependency of `near-api-js`? If so i think we can remove this as an explicit dependency